### PR TITLE
Invite email multi site 

### PR DESCRIPTION
**Pull request by MyPyDavid**
For #576 
* [x] add the tests

### CODE REVIEW

File: rdmo/projects/utils.py [hunk 1]
```diff
@@ -128,3 +132,16 @@ def save_import_tasks(project, tasks):
...
         project.views.add(view)
+
+
+def get_invite_email_project_path(invite) -> str:
+    project_invite_path = reverse('project_join', args=[invite.token])
+    # check if the multisite environment is enabled and if the user has a member role
+    if invite.user is not None and (settings.MULTISITE or invite.user.role.member.exists()):
+        # do nothing if user is a member of the current site
+        current_site = Site.objects.get_current()
+        if not invite.user.role.member.filter(domain=current_site.domain).exists():
```
Comment (jochenklar) [inline rdmo/projects/utils.py:143]: I think here you could use `invite.user.role.member.filter(id=current_site.id).exists()`.

**Review by jochenklar — APPROVED** [review]

**Comment by jochenklar** [comment]
All fine, can be merged. Is it still WIP, @MyPyDavid ?

File: rdmo/projects/utils.py (hunk 1 above)
Comment (MyPyDavid) [reply rdmo/projects/utils.py:143]: ok, i'll change it!

**Comment by MyPyDavid** [comment]
no after your review not anymore ;) thanks @jochenklar

File: rdmo/projects/utils.py [hunk 2]
```diff
@@ -128,3 +132,16 @@ def save_import_tasks(project, tasks):
 def save_import_views(project, views):
     for view in views:
         project.views.add(view)
+
+
+def get_invite_email_project_path(invite) -> str:
+    project_invite_path = reverse('project_join', args=[invite.token])
+    # check if the multisite environment is enabled and if the user has a member role
+    if invite.user is not None and (settings.MULTISITE or invite.user.role.member.exists()):
```
Comment (jochenklar) [inline rdmo/projects/utils.py:140]: I wonder if we need the `invite.user.role.member.exists(` here. My idea was that `settings.MULTISITE` determines if we are in a multi site setup or not and `roles` have a member even when not in multi site.

Comment (MyPyDavid) [reply rdmo/projects/utils.py]: yes thank you, I've removed it now.
Each user has a `role.member`  from the model https://github.com/rdmorganiser/rdmo/blob/dd0a23244adfa9d371463e63e71f68cea35e2eef/rdmo/accounts/models.py#L163

**Pull request merged**